src/styles/_catalog.scss

```diff
@@ -14,6 +14,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 300ms, 300ms;
+    transition-property: transform, color;
```

Comment:
Just as a reminder, this can be written as one line:
transition: transform 0.3s, color 0.3s;

Reply:
Okay. fixed.